src/Reader/Feed/AbstractFeed.php

```diff
@@ -268,7 +270,7 @@ public function getExtension($name)
         return null;
     }
 
-    protected function loadExtensions()
+    protected function loadExtensions(): void
```

Comment:
Please do not add php typehints as this will is a BC break unless the class is final.